src/Cake.SqlServer/SqlServerAliases.cs

```diff
@@ -317,7 +317,7 @@ public static SqlConnection OpenSqlConnection(this ICakeContext context, String
         ///             SetCommandTimeout(60);
         ///             using (var connection = OpenSqlConnection(@"Data Source=(LocalDb)\v12.0;Initial Catalog=MyDatabase"))
         ///             {
-        ///                 ExecuteSqlCommand(connection, "...");
+        ///                 SetSqlCommandTimeout(connection, "...");

```

Comment:
If you like, you are able to reuse this same PR by reopening it, amending your previous commit, and force pushing to the same branch.